index.html

```diff
@@ -203,6 +203,41 @@ <h1 class="introduction">I’m Mariane Glad to see you!</h1>
       <li class="topic item">Skills</li>
     </ul>
   </div>
+
+  <footer class="footer">
+    <div class="contact-me">
+      <h2>Contact me</h2>
+      <p>
+        If you have an application you are interested in developing, a feature that you need built or a project that
+        needs coding. I’d love to help with it
+      </p>
+    </div>
+
+    <div class="contact-forms">
+      <form class="message-form" action="https://formspree.io/f/form_id" method="post">
+        <ul>
+          <li class="input-li">
+            <input type="text" name="text" id="text-input" placeholder="Yeremias" maxlength="30" required>
+          </li>
+          <li class="input-li">
+            <input type="email" name="email" id="mail" placeholder="natajay14@gmail.com" required>
+          </li>
+          <li class="text-area-li">
+            <textarea name="msg" id="msg" maxlength="500" required>Write your messages here</textarea>
```

Comment:
- [ ] Please remove the content you have put in the `textarea` tag (i.e: Write your messages here) Instead add the `placeholder` attribute there in textarea tag then put that same text in that placeholder. This is because textarea input needs to correct data from the user, therefore It is not appropriate to hardcode the content inside it 😃 👍.